tests/cupy_tests/random_tests/test_distributions.py

```diff
@@ -539,7 +539,7 @@ def test_power(self, a_dtype, dtype):
     @cupy.testing.for_float_dtypes('dtype', no_float16=True)
     @cupy.testing.for_float_dtypes('a_dtype')
     def test_power_for_zero_a(self, a_dtype, dtype):
```

Comment:
Why not rename this test?
```suggestion
     def test_power_for_one_a(self, a_dtype, dtype):
```

Reply:
BTW, why is this test needed?

Reply:
Oh I have no idea. Maybe we can just remove this test?